runtime/compiler/z/runtime/Math.m4

```diff
@@ -831,7 +831,9 @@ ifdef({ASM_J9VM_JIT_32BIT_USES64BIT_REGISTERS},{dnl
 
 ifdef({J9ZOS390},{dnl
 ifdef({TR_HOST_64BIT},{dnl
+
 ZZ 64bit XPLINK doesn't need call descriptors
+
```

Comment:
I don't think these blank lines should be added. If I read this right, it will result in an extra blank line for 64-bit builds (which we explicitly don't want).

Reply:
Removed, also from the other similar case in `PicBuilder.m4`